src/core/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ target_sources(mrcpp
     ${CMAKE_CURRENT_SOURCE_DIR}/ObjectCache.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/QuadratureCache.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/ScalingBasis.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/SchrodingerEvolution_CrossCorrelation.cpp
```

Comment:
Same stuff for file names: `SchrodingerEvolutionCrossCorrelation`

Reply:
I used to read "The practice of programming" when I was young. It is not a good idea to have long names combined from words starting with capital letters. In this case `_` separates prefix, which makes it better for readability in my humble opinion.

Reply:
There is no right or wrong when it comes to conventions. There is a style that is adopted in a given code. And the style adopded in mrchem/mrcpp is `MyClass myObject` so it should be used throughout the code. If everyone starts adopting a different style the code becomes a mess :-)

Reply:
What about examples/poisson.cpp line 45?
`mrcpp::PoissonOperator P(MRA, prec);`
Shouldn't it be `mrcpp::PoissonOperator Poisson(MRA, prec);` or `mrcpp::PoissonOperator operatorPoisson(MRA, prec);`.
IPersonallyDislikeThisConventionSoMuchBecauseICannotReadItMyselfNotJustBecauseIThinkItIsJustBetter :)

Reply:
I'm sure there are places where the convention is not respected, and that should ideally be changed. That's not a good reason avoid using it. I think this is better discussed tomorrow at th meeting so we are all on the same page.